tensorflow/lite/delegates/gpu/gl/kernels/elementwise.cc

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 #include "tensorflow/lite/delegates/gpu/gl/kernels/elementwise.h"
 
 #include <string>
+#include <iostream>
```

Comment:
we don't want to depend on iostream.

please remove this line and remove the std::cerr below.  that's bad leftover from someone's debugging.